pkg/meta/base.go

```diff
@@ -230,6 +246,90 @@ func (m *baseMeta) checkRoot(inode Ino) Ino {
 	}
 }
 
+func (m *baseMeta) countDirUsage(ctx Context, ino Ino) (space, inodes uint64, err syscall.Errno) {
+	var entries []*Entry
+	if err = m.en.doReaddir(ctx, ino, 1, &entries, -1); err != 0 {
+		return
+	}
+	for _, e := range entries {
+		if string(e.Name) == "." || string(e.Name) == ".." {
```

Comment:
`doReaddir` will not contain these two entries.

Reply:
Removed